website/docs/docs/building-a-dbt-project/building-models/python-models.md

```diff
@@ -615,7 +695,9 @@ storage.objects.create
 storage.objects.delete
 ```
 
-**Installing packages:** Google recommends installing Python packages on Dataproc clusters via initialization actions:
+**Installing packages:** If you are using a Dataproc Cluster (as opposed to Dataproc Serverless), you can add third-party packages while creating the cluster.
```

Comment:
I saw them call this "Dataproc on Compute Engine" on their docs, which sounds ridicuous